net/unbound/files/unbound.init

```diff
@@ -40,6 +40,7 @@ start_service() {
   procd_open_instance "unbound"
   procd_set_param command $PROG -d -c $UB_TOTAL_CONF
   procd_set_param respawn
+  procd_set_param file "$UB_TOTAL_CONF" "$UB_SRV_CONF" "$UB_EXT_CONF"
```

Comment:
Would be nice to have some commit description, what is the reason, why it needs to be added now.
Based on https://openwrt.org/submitting-patches#submission_guidelines